docs/general/ddl/data-types.rst

```diff
@@ -347,6 +351,66 @@ In these expressions, the desired time zone is specified as a string
 The scalar function :ref:`TIMEZONE <scalar-timezone>` (zone, timestamp) is
 equivalent to the SQL-conforming construct timestamp AT TIME ZONE zone.
 
+.. _time-data-type:
+
+time with time zone
+-------------------
+
+The time type consist of time followed by an optional time zone.
+
+``timetz`` is a valid alias for `time with time zone`.
```

Comment:
maybe add it to the table with aliases https://github.com/crate/crate/pull/9915/files#diff-2e33689810bcdd9ac3b65fd85893a637L1176